CHANGELOG.rst

```diff
@@ -6,6 +6,15 @@ CHANGELOG
 0.9
 ---
 
+* ``SolrQuerySet`` now supports Solr grouping via new `group`
+  method and `GroupedResponse`
+* New class method `prep_index_chunk` on ``Indexable`` class, to support
+  prefetching related objects when iterating over Django querysets for indexing
+* Include django view mixins in sphinx documentation  
```

Comment:
Big, useful, well-generalized stuff! Woohoo!

Reply:
Thank you! Now I just have to make sure the tests are passing (our version matrix was a little out of date...)